provider/provider_test.go

```diff
@@ -89,6 +90,50 @@ func TestReportUpgradeCoverage(t *testing.T) {
 	providertest.ReportUpgradeCoverage(t)
 }
 
+func TestEksClusterInputValidations(t *testing.T) {
```

Comment:
Things is executed as part of the upgrade tests right now in CI. What do you think about moving it to it's own job?

Reply:
Hmm, interesting, I thought that we're trying to put tests that are not useful as examples to end-users now under `provider/*` instead of under `examples/*`, I can backlog cleaning this up.

Reply:
https://github.com/pulumi/pulumi-eks/issues/1257